urwid/_posix_raw_display.py

```diff
@@ -371,7 +382,7 @@ def get_cols_rows(self) -> tuple[int, int]:
         y, x = super().get_cols_rows()
         with contextlib.suppress(OSError):  # Term size could not be determined
             if hasattr(self._term_output_file, "fileno"):
-                buf = fcntl.ioctl(self._term_output_file.fileno(), termios.TIOCGWINSZ, " " * 4)
```

Comment:
This is buggy code: ioctl expect Buffer protocol, which includes `bytes`, but not `str`